package.json

```diff
@@ -146,6 +147,7 @@
     "focus-trap-react": "^10.0.2",
     "graphql": "^16.5.0",
     "graphql-tag": "^2.12.6",
+    "http-error-classes": "^1.0.1",
```

Comment:
I do not understand this choice of library:

1. As I see it we do not use it for anything special apart from a single error type with some predefined fields. 
2. We do not use all the predefined error classes. Any they do not have much relevance to us.
3. It is not very widely used and/or from a well-known community actor.

Based on your work with `FetcherError` it seems we could have done this ourselves or used [a more popular package for managing custom error types](https://www.npmjs.com/search?q=custom%20errors).

Reply:
You're right. I have replaced the class with our own error class. The key concepts of the class is the status code and the possibility of giving it a context.